packages/datagrid/src/types/index.ts

```diff
@@ -18,7 +23,11 @@ export type Sample = {
 };
 
 export type AvroField = {
+	name: string;
+	doc: string;
+	[CUSTOM_FIELD_DISPLAY_NAME]?: string;
 	type: any;
+	[otherProp: string]: unknown;
```

Comment:
I don't get the point of this one

Reply:
I don't master well this schema, but apparently is it possible to have some properties like
```
'@talend-quality@': {
	'0': 0,
	'1': 38,
	'-1': 62,
	total: 100,
}
```
So I add this generic properties to handle them.

Example in the UT :
https://github.com/Talend/ui/pull/4452/files#diff-a9c7df2f71f4da3a9ad08343ecbb5b00561565b6b29275d192551fd652ea4018R20-R25